### PR TITLE
feat(PictureSlider):gallery displays images from curation

### PR DESCRIPTION
**Pull request by ashemesh**
widget: https://github.com/YotpoLtd/yotpo-widget/pull/744/
ds: https://github.com/YotpoLtd/yotpo-ds/pull/87

### CODE REVIEW

File: sass/core.scss [hunk 1]
```diff
@@ -1602,6 +1602,160 @@
...
+      .yotpo-header-element {
+        padding-top: 2px;
+        display: inline-block;
+        vertical-align: top;
+        .y-label{
+          min-width: 0px;
+          padding: 0px;
+          font-size: 12px;
+          vertical-align: top;
+          &.yotpo-review-date{
```
Comment (livnekes) [inline sass/core.scss]: @ashemesh space

File: sass/core.scss [hunk 2]
```diff
@@ -1602,6 +1602,160 @@
...
+          color: $primary
+        }
+        .yotpo-instagram-location {
+          padding-left: 14px;
+          padding-top: 8px;
+          font-size: 14px;
+          color: $primary-l1;
+        }
+      }
+      .yotpo-header-actions{
```
Comment (livnekes) [inline sass/core.scss]: @ashemesh space

**Comment by livnekes** [comment]
@gadinehab this is ok ⭐

**Comment by gadinehab** [comment]
@NetanelMoshkovitz ok
with https://github.com/YotpoLtd/yotpo-widget/pull/744

**Pull request merged**